### PR TITLE
Issue 46942: Link reports are completely broken

### PR DESCRIPTION
**Pull request by labkey-sweta**
#### Rationale
Automation test for https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=47004

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

**Review by labkey-danield — APPROVED** [review]

**Review by labkey-tchad — COMMENTED** [review]
Update the existing `NonStudyReportsTest.doLinkReportTest` to use this URL.
Also, don't make the verification so specific to the current text on `labkey.com`. That stuff is fairly likely to change at some point.

**Comment by labkey-sweta** [comment]
> Update the existing `NonStudyReportsTest.doLinkReportTest` to use this URL. Also, don't make the verification so specific to the current text on `labkey.com`. That stuff is fairly likely to change at some point.

Does it make sense to keep one test with url linking to external website and one with url linking to internal.

File: src/org/labkey/test/tests/LinkedReportTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        setFormElement(Locator.name("viewName"), REPORT_NAME);
+        setFormElement(Locator.name("linkUrl"), LINK_REPORT_URL);
+        clickButton("Save");
+        waitForText("Manage Views");
+
+        waitAndClick(Locator.linkWithText(REPORT_NAME));
+        switchToWindow(1);
+
+        waitForElement(Locator.linkWithText("Solutions"));
+        Assert.assertTrue("Linked report navigated to incorrect external link", isElementPresent(Locator.linkWithText("Get a Demo")));
```
Comment (labkey-tchad) [inline src/org/labkey/test/tests/LinkedReportTest.java]: Still too specific. Just checking the URL should be sufficient.

**Review by labkey-tchad — APPROVED** [review]
You're probably right. There's enough difference between link reports to external and internal resources that we should test them both (considering there was an issue with one but not the other).

File: src/org/labkey/test/tests/LinkedReportTest.java (hunk 1 above)
Comment (labkey-sweta) [reply src/org/labkey/test/tests/LinkedReportTest.java]: using getURL after swtichtowindow
is throwing Bad location from selenium tester: about:blank
any work around?

**Pull request merged**